src/components/inputs/preview.vue

```diff
@@ -1,26 +1,25 @@
 <template>
-    <div>
-        <h3>CBD input components</h3>   
-        
-        <div class="row">
-            <div class="col-12">
-                <div class="card">
-                    <div class="card-header">
-                        Checkbox
-                    </div>
-                    <div class="card-body"> 
-                        <div class="row">
-                            <div class="col-6">
-                                <checkbox  v-model="isChecked">
-                                    <template #label>
-                                        <!-- Content for the label slot -->
-                                        This is the label slot
-                                    </template>
-                                </checkbox>
-                            </div>
-                            <div class="col-4">
-                                <strong>sample code</strong>
-                                <textarea rows="10" cols="80" class="bg-purple text-white"> 
+  <div>
+    <h3>CBD input components</h3>
+
+    <div class="row">
+      <div class="col-12">
+        <div class="card">
+          <div class="card-header">Checkbox</div>
+          <div class="card-body">
+            <div class="row">
+              <div class="col-6">
+                <checkbox v-model="isChecked">
+                  <template #label>
+                    <!-- Content for the label slot -->
+                    This is the label slot
+                  </template>
+                </checkbox>
+              </div>
+              <div class="col-4">
+                <strong>sample code</strong>
+                <textarea rows="10" cols="80" class="bg-purple text-white">
```

Comment:
please do not commit code that you have not edited

Reply:
This update is caused by the autoformatter, there is no change to the code.